styleguide/components/51-templates/styleguide/styleguide.twig

```diff
@@ -50,34 +49,22 @@
         <li>
           Organisms
           <ol>
+            <li><a href="#styleguide-footer">Footer</a></li>
```

Comment:
No single items list (spotted this rule in the documentation yesterday)

Reply:
It's part of the table of contents...